site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -150,5 +150,11 @@ The `spec` field is the actual `GameServerAllocation` specification, and it is c
 - `metadata` is an optional list of custom labels and/or annotations that will be used to patch
   the game server's metadata in the moment of allocation. This can be used to tell the server necessary session data
 
-[gameserverselector]: {{% ref "/docs/Reference/agones_crd_api_reference.html#allocation.agones.dev/v1.GameServerSelector"  %}}
+Each `GameServerAllocation` also considers [namespaces][namespace] when allocating GameServers. Agones will only
+attempt to allocate GameServers from the specified namespace. The [namespace][namespace] can be specified outside of
```

Comment:
the second use of namespaces doesn't also need a link out to the k8s docs.

Reply:
Is it "can" or "must" here? I think through your testing you found that the namespace _cannot_ be specified in the yaml file and _must_ be provided out of band (flag to kubectl or url parameter).

Reply:
Yeah, the namespace cannot be specified in the yaml file but I wrote "can" instead of "must" because the namespace can be excluded completely in the flag to kubectl, in which case the namespace will default to "default". Do you think it's still better if it's "must"?

Reply:
I see your point. If specified, it must be out of band. But it need not be specified _if you are using the CLI_ (it will default to `default`). 

I suppose since this is how all k8s resources work, this doesn't need to be too explicit. Let's leave it as "can".